components/graph/git-node.js

```diff
@@ -183,7 +184,6 @@ GitNodeViewModel.prototype.showRefSearchForm = function(obj, event) {
     $(this).autocomplete('search', $(this).val());
   }).data("ui-autocomplete")._renderItem = function (ul, item) {
     return $("<li></li>")
-      .data("item.autocomplete", item)
```

Comment:
This is already taken care of here https://github.com/jquery/jquery-ui/blob/809f29efa79c3c9aba95e6d7ffef41f567cda3a5/ui/widgets/autocomplete.js#L562-L564